pallets/sponsorship/src/lib.rs

```diff
@@ -157,19 +157,42 @@ pub mod pallet {
 	#[pallet::generate_deposit(pub(super) fn deposit_event)]
 	pub enum Event<T: Config> {
 		/// Event emitted when a new pot is created.
-		PotCreated { pot: T::PotId },
+		PotCreated {
+			pot: T::PotId,
+			sponsor: T::AccountId,
+			sponsorship_type: T::SponsorshipType,
+			fee_quota: BalanceOf<T>,
+			reserve_quota: BalanceOf<T>,
+		},
 		/// Event emitted when a pot is removed.
 		PotRemoved { pot: T::PotId },
 		/// Event emitted when a pot is updated.
-		PotUpdated { pot: T::PotId },
+		PotUpdated {
+			pot: T::PotId,
+			fee_quota: BalanceOf<T>,
+			reserve_quota: BalanceOf<T>,
+		},
 		/// Event emitted when a pot is updated.
-		PotSponsorshipTypeUpdated { pot: T::PotId },
+		PotSponsorshipTypeUpdated {
+			pot: T::PotId,
+			sponsorship_type: T::SponsorshipType,
+		},
 		/// Event emitted when user/users are registered indicating the list of them
-		UsersRegistered { pot: T::PotId, users: Vec<T::AccountId> },
+		UsersRegistered {
+			pot: T::PotId,
+			users: Vec<T::AccountId>,
+			fee_quota: BalanceOf<T>,
+			reserve_quota: BalanceOf<T>,
+		},
 		/// Event emitted when user/users are removed indicating the list of them
 		UsersRemoved { pot: T::PotId, users: Vec<T::AccountId> },
 		/// Event emitted when fee_quota or reserve_quota or both are updated for the given list
-		UsersLimitsUpdated { pot: T::PotId, users: Vec<T::AccountId> },
+		UsersLimitsUpdated {
+			pot: T::PotId,
+			users: Vec<T::AccountId>,
+			fee_quota: BalanceOf<T>,
+			reserve_quota: BalanceOf<T>,
+		},
 		/// Event emitted when a sponsor_me call has been successful indicating the reserved amount
 		Sponsored { paid: BalanceOf<T>, repaid: BalanceOf<T> },
```

Comment:
What `repaid` means?

Reply:
@Douglasacost `paid` is how much was borrowed from the sponsor at the beginning of this call and `repaid` means how much debt was payed off from the user's account at the end of this call.